atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/SweepBatchConfigTest.java

```diff
@@ -15,57 +15,54 @@
  */
 package com.palantir.atlasdb.sweep;
 
-import org.junit.Rule;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
 import org.junit.Test;
-import org.junit.rules.ExpectedException;
 
 @SuppressWarnings("CheckReturnValue")
 public class SweepBatchConfigTest {
-    @Rule
-    public ExpectedException exception = ExpectedException.none();
 
     @Test
     public void canCreateConfig() {
-        ImmutableSweepBatchConfig.builder()
-                .maxCellTsPairsToExamine(1)
-                .candidateBatchSize(1)
-                .deleteBatchSize(1)
-                .build();
+        assertThat(ImmutableSweepBatchConfig.builder()
```

Comment:
there is also
```java
assertThatCode(() -> ...)
    .doesNotThrowAnyException();
```

which may be more fitting for places like this (although checking it isn't null isn't wrong, exactly)